NomaiVR/UI/Dialogue.cs

```diff
@@ -11,21 +11,23 @@ internal class Dialogue : NomaiVRModule<Dialogue.Behaviour, Dialogue.Behaviour.P
 
         public class Behaviour : MonoBehaviour
         {
+            private static GameObject _uiGameObject;
             private static Transform _canvasTransform;
             private static Transform _attentionPoint = null;
             private const float _dialogeRenderSize = 0.0015f;
 
             internal void Start()
             {
-                _canvasTransform = GameObject.Find("DialogueCanvas").transform;
+                _uiGameObject = GameObject.FindWithTag("DialogueGui");
+                _canvasTransform = _uiGameObject.transform;//GameObject.Find("DialogueCanvas").transform;
 
                 _canvasTransform.localScale *= _dialogeRenderSize;
 
                 // Prevent dialogue box from flying off after a while.
-                _canvasTransform.parent = new GameObject().transform;
-                _canvasTransform.parent.gameObject.AddComponent<FollowTarget>().target = Locator.GetPlayerTransform();
+                _canvasTransform.parent = Locator.GetPlayerTransform();
+                //_canvasTransform.parent.gameObject.AddComponent<FollowTarget>().target = Locator.GetPlayerTransform();
```

Comment:
I think there was some reason why I had to use a FollowTarget here instead of just normal parenting? It was breaking something. Should have left a comment explaining it. I'll investigate a bit to see why that was there.

Reply:
Yep these changes come from a test I was making I forgot to clean them up for the PR.
It seems to work fine for now, I'll revert them for this PR since they are out of scope.

Reply:
It does actually seem to work fine and is obviously much cleaner. But I agree, better leave as it was to avoid breaking something 🤷‍♂️